modules/nextflow/src/main/groovy/nextflow/spack/SpackCache.groovy

```diff
@@ -299,7 +299,7 @@ class SpackCache {
     }
 
     @PackageScope
-    void runCommand( String cmd ) {
+    int runCommand( String cmd ) {
```

Comment:
Yep, I think the problem was the test. But the return value is not used anywhere. so why adding it?

Reply:
isn't is used in all the unit tests as exitcode, for logical (true/false) checks? 

e.g. see your second edit in https://github.com/nextflow-io/nextflow/commit/868374d3d591113a6fa3364d4e520d2ef3374aaf#diff-86b11d6f83a372628c1957303e638eccaa9583281cdb8657845b1222b6ff570a